exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/cloud/01_import_data.ipynb

```diff
@@ -2,41 +2,23 @@
  "cells": [
```

Comment:
<p>Why not let the user enter the credentials in with a UI? (see the sagemaker/sme_init.ipynb). Separate ticket perhaps.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/114/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Default empty values are working for public S3 buckets. So, this notebook will work without any configuration of S3 credentials.</p><p>They are needed only if user wants to experiment with their own data on private bucket. But moving those options into configuration </p><p>UI is a good idea, will add a ticket</p>